lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -592,31 +583,52 @@ sub variant_exists {
         next;
     }
 
-    # Precise match for snv or small indels - end does not make a difference
-    if ($sv == 0 && ($seq_region_start != $start_pos) && ($seq_region_end != $end_pos)) {
-      next;
+    # Type of query
+    my $bracket_query;
+    if($start_pos != $start_max_pos || $end_pos != $end_min_pos) {
+      $bracket_query = 1;
+    }
+
+    # Match for snv or small indels
+    # Range query is supported - this type of query only requires the ref allele
+    if($sv == 0) {
+      next if (!$range_query && $seq_region_start != $start_pos && $seq_region_end != $end_pos);
+      # Range query
+      next if ($range_query && $seq_region_start < $start_pos || $seq_region_end > $end_pos);
```

Comment:
Could this be moved to the `if ($sv == 0)` statement in line 611 to put everything about SNVs together?

Reply:
It can!